modules/search/class.jetpack-search.php

```diff
@@ -462,7 +511,28 @@ public function filter__posts_pre_query( $posts, $query ) {
 			return $posts;
 		}
 
-		// If no results, nothing to do
+		if ( count( $this->search_result['results']['hits'] ) ) {
+			// If results, check for a "did you mean" suggestion.
+			add_filter( 'loop_start', array( $this, 'filter__show_did_you_mean' ) );
```

Comment:
Interesting. I feel like we should have some threshold for when we show this. Figuring that out is non-trivial of course, and maybe it is already controlled by the way the suggest is written?

We'll probably want some filters so that individual sites can customize this, but this is all probably low priority for the moment.

Reply:
Sounds good.